libjwt/jwt-openssl.c

```diff
@@ -364,18 +371,27 @@ int jwt_verify_sha_pem(jwt_t *jwt, const char *head, const char *sig_b64)
 		return EINVAL;
 	}
 
-	sig = jwt_b64_decode(sig_b64, &slen);
-	if (sig == NULL)
-		VERIFY_ERROR(EINVAL);
-
-	bufkey = BIO_new_mem_buf(jwt->key, jwt->key_len);
-	if (bufkey == NULL)
-		VERIFY_ERROR(ENOMEM);
-
-	/* This uses OpenSSL's default passphrase callback if needed. The
-	 * library caller can override this in many ways, all of which are
-	 * outside of the scope of LibJWT and this is documented in jwt.h. */
-	pkey = PEM_read_bio_PUBKEY(bufkey, NULL, NULL, NULL);
+    sig = jwt_b64_decode(sig_b64, &slen);
```

Comment:
You've changed indentation for lines you didn't have code changes for. Also, all of your code does not match the current indentation (tab indents, not spaces).

Please redo your indents to match the rest of the code base and keep you changes to a minimum (do not re-indent code you aren't changing).

Reply:
I have changed the indentation to use tabs instead of spaces.